Example/VOKUtilities/AnotherViewController.m

```diff
@@ -0,0 +1,36 @@
+//
+//  AnotherViewController.m
+//  VOKUtilities
+//
+//  Created by Brock Boland on 1/24/17.
+//  Copyright © 2017 Vokal. All rights reserved.
+//
+
+#import "AnotherViewController.h"
+
+@implementation AnotherViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+    // Add a dummy label to the view, for UI test
+    UILabel *label = [[UILabel alloc] init];
+    label.text = @"This is a label from code";
+    [self.view addSubview:label];
+    [self.view addConstraint:[NSLayoutConstraint constraintWithItem:label
+                                                          attribute:NSLayoutAttributeCenterX
+                                                          relatedBy:NSLayoutRelationEqual
+                                                             toItem:self.view
+                                                          attribute:NSLayoutAttributeCenterX
+                                                         multiplier:1
+                                                           constant:0]];
+    [self.view addConstraint:[NSLayoutConstraint constraintWithItem:label
+                                                          attribute:NSLayoutAttributeCenterY
+                                                          relatedBy:NSLayoutRelationEqual
+                                                             toItem:self.view
+                                                          attribute:NSLayoutAttributeCenterY
+                                                         multiplier:1
+                                                           constant:0]];
```

Comment:
Does the example project have to work back to iOS 7?  Otherwise, maybe layout anchors?